### DISCUSSION TITLE
Is it possible to access OpenTelemetry SDK via COM+ (Classic ASP)?

### DISCUSSION THREAD

**Discussion by TigerC10** [open-telemetry/opentelemetry-dotnet#3874] [Q&A]
Trying to see if I can instrument a Classic ASP app for distributed traces/spans.  I know that in Classic ASP COM objects can bridge the .NET divide, but not all shared libraries can be accessed via COM+ so wondering if otel is (or could be).

**Comment by seelaz** [comment]
Have you come to a solution by this approach?

**Reply by TigerC10** [reply]
No.
